dashboard/app/views/api/hoc_courses.html.haml

```diff
@@ -1,25 +1,20 @@
 - page_mode = PageMode.get(request)
+- if page_mode == "feature-mc"
+  - first_tutorial = Script::MINECRAFT_NAME
+  - second_tutorial = Script::STARWARS_NAME
+- elsif page_mode == "feature-starwars"
```

Comment:
is it possible that when you merge this, page mode won't be "feature-starwars" or "feature-minecraft"? would it be better to have a true else case?